task/buildah/0.2/buildah.yaml

```diff
@@ -309,10 +311,12 @@ spec:
         BUILDAH_ARGS+=("--skip-unused-stages=false")
       fi
 
+      VOLUME_MOUNTS="$VOLUME_MOUNTS_FROM_ENV"
+
```

Comment:
Rename it to just `ADDITIONAL_VOLUME_MOUNTS`, which could imply that it can be specified/customized additionally by someone else. The part "_FROM_ENV" looks useless because it is clear it's an environment variable.

`VOLUME_MOUNTS` should be commented here.

Reply:
Makes sense